configure.ac

```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle],[5.1.4],[http://bugs.launchpad.net/drizzle],[libdrizzle],[http://drizzle.org/])
```

Comment:
Hmm, yes that the question. Probably it could be a variable defined using a git tag or some other dynamically updated version number...